src/modules/Host.rb

```diff
@@ -358,12 +358,23 @@ def set_names(address, names)
   def load_hosts
     return false if SCR.Read(path(".target.size"), CFA::Hosts::PATH) <= 0
 
-    @hosts = CFA::Hosts.new
-    @hosts.load
+    begin
+      @hosts = CFA::Hosts.new
+      @hosts.load
+
+      # save hosts to check for changes later
+      @hosts_init = CFA::Hosts.new
+      @hosts_init.load
+    rescue IOError, SystemCallError, RuntimeError => error
```

Comment:
I would prefer rescuing their common superclass, which is StandardError.

I know I said "RuntimeError is too generic" in the original PR, but that was in the context of a global Yast handler for such things where I would introduce a GarbageDataError. And I forgot to do it myself when [changing CFA](https://github.com/config-files-api/config_files_api/pull/23/files#diff-53b7f088c9b3245fd0e594323ab213e8R377)

In this context, enumerating the exceptions is confusing to me as a reader, wondering whether we are letting some other exceptions slip through intentionally. Are we? Add a comment if yes, use StandardError if no.

Reply:
added comment